visualization/include/pcl/visualization/area_picking_event.h

```diff
@@ -53,7 +53,7 @@ namespace pcl
       public:
         AreaPickingEvent (int nb_points, const std::map<std::string, Indices>& cloud_indices)
           : nb_points_ (nb_points)
-          , cloud_indices_ (cloud_indices)
+          , cloud_indices_ (std::move(cloud_indices))
```

Comment:
Use `std::map<> cloud_indices` aka by value parameter in the ctor to ensure you're not moving out a value that could be used by another user somewhere